FaunaDB.Client/Utils/CheckLatestVersion.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FaunaDB.Client.Utils
+{
+    public class CheckLatestVersion
+    {
+        private const string packageName = "FaunaDB.Client";
+        public static bool? AlreadyChecked { get; set; } = null;
```

Comment:
Why we can't just set it to `false` as a default value instead of using a nullable type?